kr-core/src/main/clojure/edu/ucdenver/ccp/kr/sparql.clj

```diff
@@ -243,11 +244,43 @@
        (sparql-query-body optional-clauses)
        " }\n "))
 
+(defn filter-body [filter-clauses]
+  (str " FILTER ( "
+       (operator-body filter-clauses)
+       " )\n "))
+
+(defn bind-body [bind-clauses]
+  (str " BIND "
+       (operator-body bind-clauses)
+       " \n "))
+
+(defn iri-body [iri-clauses]
+  (str " IRI "
+       (operator-body iri-clauses)
+       " \n "))
+
+(defn concat-body [concat-clauses]
+  (str " CONCAT "
+       (operator-body concat-clauses)
+       " \n "))
+
+(defn strafter-body [strafter-clauses]
+  (str " STRAFTER "
+       (operator-body strafter-clauses)
+       " \n "))
+
+
+
+
```

Comment:
Does the specificity of the method name add enough value to justify this duplication?  I feel as though `(operator-body "IRI" [clauses])` isn't significantly less readable than `(iri-body [clauses])`

Alternatively, we could create a macro that would enable something like `(defoperator iri)` to automatically define the appropriate function without the duplication.